### PR TITLE
Issue 8233 | Tried to clarify wording regarding custom errorHandler cfg

### PR DESCRIPTION
**Pull request by zanella**
- https://github.com/playframework/playframework/issues/8233

### CODE REVIEW

File: documentation/manual/working/scalaGuide/main/http/ScalaErrorHandling.md [hunk 1]
```diff
@@ -36,7 +36,14 @@ If you're using runtime dependency injection (e.g. Guice), the error handler can
 
 @[root](code/ScalaErrorHandling.scala)
 
-If you don't want to place your error handler in the root package, or if you want to be able to configure different error handlers for different environments, you can do this by configuring the `play.http.errorHandler` configuration property in `application.conf`:
+If you place your error handler in the root package (i.e. package-less) Play will use it by default.
```
Comment (marcospereira) [inline documentation/manual/working/scalaGuide/main/http/ScalaErrorHandling.md]:
```suggestion
If you place your error handler in the root package (i.e. package-less) and with name `ErrorHandler`, Play will use it by default.
```

**Comment by marcospereira** [comment]
We can backport this to 2.7.x and 2.6.x.

File: documentation/manual/working/scalaGuide/main/http/ScalaErrorHandling.md (hunk 1 above)
Comment (zanella) [reply documentation/manual/working/scalaGuide/main/http/ScalaErrorHandling.md]: Clarification added

**Comment by marcospereira** [comment]
Backport to 2.7.x: 174a7f03fbea09318ca8b896260e70f21f5b868d

**Pull request merged**